Whiskey/Whiskey.psd1

```diff
@@ -153,7 +153,20 @@
 
             # ReleaseNotes of this module
             ReleaseNotes = @'
+* NUnit3 task supports running NUnit2 tests and uses version 3.10 of ConsoleRunner.

```

Comment:
These are the release notes from the previous `0.41.0` release. The release notes should simply have:
```
* Fixed: `Version` task uses the last matching item in the prerelease label branch map instead of the first match.
* <Insert your release notes here>
```